src/blocks/EventAccordion/utils/text.js

```diff
@@ -0,0 +1,3 @@
+const formatOnSaleText = text => text.replace(":", ":\n");
```

Comment:
We should let devs handle this formatting before they pass the prop to the component